core/code/chat.js

```diff
@@ -378,10 +378,10 @@ window.chat.parseMsgData = function(data) {
       nick = ent[1].plain.replace(/: $/, ''); // cut “: ” at end
       break;
 
-    case 'PLAYER': // automatically generated messages
-      nick = ent[1].plain;
-      team = ent[1].team === 'RESISTANCE' ? TEAM_RES : TEAM_ENL;
-      break;
+      case 'PLAYER': // automatically generated messages
```

Comment:
```suggestion
    case 'PLAYER': // automatically generated messages
```
In this block, please remove the extra spaces

Reply:
Actually, the whole switch needed to have one indendation level added - this was the only correctly indended case 🙈